### ISSUE TITLE
PNP Search Box is not searching when I'm define Documents in Result Search ID

### ISSUE THREAD

**Issue by AnderOllive** [microsoft-search/pnp-modern-search#1294]
**Version used**
Ex: 4.3

**Question**
I'm using in PNP Results option:
Query Template= {searchTerms}*
Result source ID: Documents

When connect PNP Search Box, i can't make search, this search box is not working. Details, search box, only is not working when select "Result search ID: Documents"
I tested in 2 tenants, the same behavior happens in both.
I want to know if when we using this options we have to configure something more.

**Screenshots**

![image](https://user-images.githubusercontent.com/88772391/133286897-230bf64f-5251-431a-b741-d7be1dbc00b3.png)
![image](https://user-images.githubusercontent.com/88772391/133287100-3a927e99-8c81-4305-b143-191c6a405318.png)

Search box

![image](https://user-images.githubusercontent.com/88772391/133287896-d23f1c4a-182e-41ed-9526-bea7b880a2a3.png)
![image](https://user-images.githubusercontent.com/88772391/133287971-7c92f77b-d229-41d4-8b58-64d8ef7a6ed8.png)

**Comment by wobba** [comment]
Please see if this solves your issue: https://microsoft-search.github.io/pnp-modern-search/scenarios/create-simple-search-page/

**Comment by AnderOllive** [comment]
@wobba  This was exactly thas I did, but how to can see above in images, the problem occur just when we select the option 'documents' in result search id. If i choose other options, for example 'localsharepoint' the search box work normally.

**Comment by FranckyC** [comment]
Hi @AnderOllive, I think this is because the Documents result source and query template are not compatible/cumulative. If the 'Documents' source queyr doesn't contain `{searchTerms}`, your keywords are never picked up and query template is ignored.

To bypass this, you can:

- Create your own result source as a copy of the 'Documents' one including `{searchTerms}`
- Build your query directly in the **Query template**.

**Comment by AnderOllive** [comment]
I guess that I'll need to create a query, but I'm using filter too. So, better I not use search box in this case. More one time, tks for your help and patience. 👍

**Comment by FranckyC** [comment]
Actually the proposed solutions achieve the same goal. It is your call and the way you prefer to manage the query in the end. Filters will work with both.

**Comment by AnderOllive** [comment]
Yes, in this moment filters working as well. thank you.

**Comment by EmelieMarkkanen** [comment]
Hi! 
I'm having the same issue getting the search box working for documents. I've set up a new search result source that is a copy of Documents, but changed the scope and added the {searchTerm} to the query as well.

The problem I've run into mainly is that my new search result source is not available to choose in the list when configuring the search result webpart. I've just named it Documents copy for now. Is there anything specific I need to do to make this available? 🤔

![pnp result source](https://github.com/user-attachments/assets/3dd2c088-8512-49ec-8bbf-e57e177b31e7)

![pnp result source list](https://github.com/user-attachments/assets/7776af7e-44ca-403e-9393-90daf6712360)
